Rubberduck.Core/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -168,9 +169,12 @@ private void InsertNewDeclaration()
             }
 
             var insertionIndex = (expression as ParserRuleContext).Start.TokenIndex;
+            var firstReferenceLine = _vbe.ActiveCodePane.CodeModule.GetLines((expression as ParserRuleContext).Start.Line, 1);
```

Comment:
`_vbe`., `ActiveCodePane`, and `CodeModule` each are a `SafeComWrapper`. Those must be wrapped in a `using` block for each reference to ensure proper release of the wrapper.

Reply:
PS I spoke a bit too hastily - `_vbe` doesn't need to be in the `using`; the other two do, though.

Reply:
Thanks, will update with a pair of ```using``` blocks